src/main/java/uk/gov/gchq/palisade/service/request/DataRequestResponse.java

```diff
@@ -74,15 +67,28 @@ public DataRequestResponse resources(final Map<LeafResource, ConnectionDetail> r
         return this;
```

Comment:
use setters

Reply:
line 64 should use setters and not check nulls

Reply:
needs the generated annotation